### DISCUSSION TITLE
Useful Web Tools

### DISCUSSION THREAD

**Discussion by daank-c** [dogecoin/dogecoin#3074] [Show and tell]
## Useful Web Tools

You know that moment when you find a Web tool and you realise it's an invaluable gem? Like for some, it might be Grammarly the keyboard extension that helps you write with confidence. Or perhaps Trello, the Web tool for organising workflow. All of which are very useful, and seemingly under-discussed.

I wanted to start this "Show and Tell" session, because we could all benefit from more tools - high-quality tools - and maybe even free ones! So, why wouldn't we share the ones we use with each other?

The idea of starting a thread for sharing our favourite tools came from me using Miro - the Online Whiteboard. Man, I couldn't
 imagine not having it - and I really don't think it's widely used. It's just as it sounds, a simple Whiteboard web app, but it enables remote teams or individuals to organise workflow, project flow, road maps, etc etc. It's also an opt-in cloud service in a sense, where two people can work on the same board at the same time. 

![Screenshot 2022-07-25 4 32 48 PM](https://user-images.githubusercontent.com/102710924/180859917-72d27055-98ea-4c11-96ab-a8885b5cd59c.png)

The reason why I think this app is so great, is it's just built well, and it's free for your first few boards. Miro has lots of templates, geometric shapes, and drawing options like object snapping. They even offer APIs if you're interested in taking your board to the next level. Obviously this isn't an endorsement or anything, I just like giving credit where I think it's due - and I hate to keep these things to myself, knowing someone else might find them useful. 

With better tools we can all work more effectively and efficiently through life, here in this repository, or even at our 9-5. 
So uhh, where da web apps at? Whose got some goodies ? 🥤😛🍿

**Comment by alamshafil** [comment]
I personally don't have much to share but one website I would like to share is [Figma] (https://www.figma.com) It's a collaborative interface designing tool. I personally use it to find templates for certain UI designs and edit them to fit my ends. 

The templates are useful for me because I'm not a UI/UX person and having a template helps to design a nice looking UI without trying to code one on the spot. 

It may not be useful for everyone but I felt like sharing this for people who may find it useful. 

![image](https://user-images.githubusercontent.com/36016500/180872616-08afc1a9-77c2-417b-b3cb-94b8282ca573.jpeg)

**Reply by daank-c** [reply]
Wow, that's actually pretty wild.

I actually have another one that you may like, @alamshafil . I use it everyday to make dumb memes, but I'm a simple man.

They call it Adobe Express and although many people might already be familiar with it, it's certainly worth mentioning. It's like Canva, but takes advantage of Adobe's expertise in the field of media creation. I would even go so far to say it's more useful, and easier to use than Canva - but to each their own.

The Web App version is expansive, and its capabilities reach much farther than a simple brochure editor. They also have an App-ified version for Mobile devices, however it runs a lighter by offering less features than they do on the Web. When you set up an account with Adobe, your projects are shared between your devices using their Creative Cloud - projects are also then accessible from Photoshop, Light room, etc., which definitely adds some useful versatility. 

The paid version of Adobe Express is obviously better, as it unlocks many features, I paid the $7/mo when I  was using it everyday, I definitely felt like the price was fair.

Lol, I should be the spokesperson, but it's seriously probably one of the best web apps I've ever used. 

![Screenshot 2022-07-25 8 37 37 PM](https://user-images.githubusercontent.com/102710924/180892939-03f5091d-393d-4caf-be9b-26da1ae2f8ae.png)

**Reply by alamshafil** [reply]
I believe this website is not as feature-packed as the one you showed above, but I believe it's cool to look at it. It is called [Photopea](https://www.photopea.com/), it's like Photoshop in the browser. For a web photo editor, I have to see it's pretty advanced since it has templates, supports .PSD files, has layers, and much more. Feel free to take a look at it!

![image](https://user-images.githubusercontent.com/36016500/181079503-0d8b045f-c7da-49e6-90f0-d0909b077b6a.png)
